Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Ammunition/Magazines/novalitec1clip.yml

```diff
@@ -0,0 +1,48 @@
+# Empty mags
+- type: entity
+  id: BaseMagazineNovaliteC1
+  name: "High Capacity Assault Clip (.20 rifle)"
```

Comment:
'magazine' instead of 'clip' please

Reply:
Renamed to Novalite En Bloc. That appears to be the historically correct name for this type of ammunition loading mechanism.